bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDescriptorImpl.java

```diff
@@ -122,7 +123,7 @@ ByteBuf getExplicitLac() {
         }
         ByteBuf entry = createLedgerFenceEntry(ledgerId);
         journal.logAddEntry(entry, (rc, ledgerId, entryId, addr, ctx) -> {
-            LOG.debug("Record fenced state for ledger {} in journal with rc {}", ledgerId, rc);
+            LOG.debug("Record fenced state for ledger {} in journal with rc {}", ledgerId, BKException.codeLogger(rc));
```

Comment:
wrap this with

   if (LOG.isDebugEnabled()) {
       LOG.debug ...
   }

codeLogger() creates new objects and in 99.9% of cases debug logging is off. This lets us bypass extra object creation.

Reply:
@dlg99: see a1aea61